base/sysimg.jl

```diff
@@ -96,7 +96,6 @@ include("subarray.jl")
 (::Type{Vector})() = Array{Any,1}(0)
```

Comment:
A much smaller number of packages use `Matrix()` or `Matrix{T}()`:

Bio,DSGE,DifferentialDynamicProgramming,Elemental,JPLEphemeris,JuliaFEM,LightGraphs,Mamba,NearestNeighbors,OpenDSSDirect,ParallelAccelerator